ercc/ercc.go

```diff
@@ -17,8 +17,9 @@
 package main
 
 import (
+	"strings"
+
```

Comment:
did you run gofmt?

Reply:
i thought make would do that for me ;-)
more seriously, running now gofmt -w ercc/ercc..go only changes white spaces in front of a comment, nothing related to import. anyway, run no gofmt on {ercc,ecc}/..../*.go which also changed some whitespaces at a few other places and in one case re-ordered import  ...